docs/staking-and-delegation/managing-stake-sdk.md

```diff
@@ -93,57 +93,133 @@ print(netuids)
 
 ## Stake
 
-The following script incrementally stakes 3 TAO into several subnets over many blocks:
+The following script incrementally stakes 1 TAO in each of the top three validators of the top three subnets:
 
 ```python
-
 import bittensor as bt
-sub = bt.Subtensor(network="test")
-wallet = bt.wallet(name="ExampleWalletName")
-wallet.unlock_coldkey()
+import time
+from bittensor import tao
 
-to_buy = [119, 277, 18, 5] # list of netuids to stake into
-increment = 0.01 # amount of TAO to stake
-total_spend = 0 # total amount of TAO spent
-stake = {} # dictionary to store the stake for each netuid
+# Initialize the wallet
+wallet = bt.wallet(name='jpe442', hotkey='taostar')
 
-while total_spend < 3:
-    for netuid in to_buy:
-        subnet = sub.subnet(netuid)
-        print(f"slippage for subnet {netuid}", subnet.slippage(increment))
-        sub.add_stake( 
-            wallet = wallet, 
-            netuid = netuid, 
-            hotkey = subnet.owner_hotkey, 
-            tao_amount = increment, 
-        )
+# Initialize the subtensor connection
+subtensor = bt.subtensor(network='test')  # Use 'main' for the main network
 
-        current_stake = sub.get_stake(
-            coldkey_ss58 = wallet.coldkeypub.ss58_address,
-            hotkey_ss58 = subnet.owner_hotkey,
-            netuid = netuid,
-        )
-        stake[netuid] = current_stake
-        total_spend += increment
-        print (f'netuid {netuid} price {subnet.price} stake {current_stake}')
-    sub.wait_for_block()
+print("Fetching subnet information...")
+all_subnets = subtensor.get_all_subnets_info()
+print(f"Retrieved {len(all_subnets)} subnets")
+
+
+# Initialize the subtensor connection
+subtensor = bt.subtensor(network='finney')  # Use 'main' for the main network
```

Comment:
you are pointing at main net now!!!

Reply:
Oops, I swear I had been point at 'test', maybe I updated the script with a copy pasta and overwrote it.